pkg/compiler/rego/rego.go

```diff
@@ -43,11 +43,19 @@ func (c *CompilerRego) Compile(pkgname string, pols *ast.Policies) (string, erro
 	compiled = append(compiled, c.compileSetDefaults("false", "allow", "deny")...)
 
 	var lineNum int
+	var out string
+	var err error
 	for idx, stmt := range pols.Statements {
 		lineNum += 1
 		switch stmt.(type) {
 		case *ast.ActionStatement:
-			out, err := c.compileStatement(stmt.(*ast.ActionStatement), lineNum)
+			out, err = c.compileStatement(stmt.(*ast.ActionStatement), lineNum)
```

Comment:
```suggestion
			out, err := c.compileStatement(stmt.(*ast.ActionStatement), lineNum)
```

Reply:
I have another suggestion. Is it looks better now?

Reply:
looks great